tests/playbooks/python_distribution.yaml

```diff
@@ -28,6 +28,10 @@
       pulp.squeezer.python_distribution:
         name: test_python_distribution
         state: absent
+    - name: Make distribution for pull-through absent
+      pulp.squeezer.python_distribution:
+        name: test_python_distribution_pull_through
+        state: absent
```

Comment:
Can you copy this to the cleanup section at the end of this file?
I think, that is where you actually confused the `python_remote` test. (Still keep the more precise regex there...)

Reply:
Fixed and pushed.